src/configuration/content-evaluator-config.ts

```diff
@@ -10,7 +10,7 @@ const openAiType = Type.Object(
     /**
      * Specific endpoint to send the comments to.
      */
-    endpoint: Type.String({ default: "" }),
+    endpoint: Type.String({ default: "https://api.openai.com/v1", pattern: /^(https?:\/\/[^\s$.?#].\S*)$/i.source }),
```

Comment:
`i.source`? What is this for?

Reply:
```suggestion
    endpoint: Type.String({ default: "https://api.openai.com/v1", pattern: /^(https?:\/\/[^\s$.?#].\S*)$/i }),
```

Reply:
@0x4007 Are you sure this compiles? the required type is a string not a regex.

Reply:
![Uploading image.png…]()

Reply:
<img width="433" alt="image" src="https://github.com/user-attachments/assets/9e8af02e-c258-4540-ab4c-10359f9d57aa">

Reply:
`source` converts the regex to its `string` representation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/source